src/domain/events/eventSearchForm/eventSearchForm.module.scss

```diff
@@ -70,6 +70,17 @@
   }
 }
 
+.checkboxWrapper {
+    align-items: center;
+    & > div > input {
+      border: none !important;
+      background: var(--color-white);
+    }
+    & > div > label {
+      color: var(--color-white) !important;
+    }
```

Comment:
If you apply a className to the checkbox component, you can overwrite these styles by using CSS variables:

```css
.className {
  --label-color: ...;
  --background-unselected: ...;
  ...
}
```

This approach has a bit clearer contract between HDS and this service.

Minor.

Might be too verbose for your taste.